english/thunder-carrots/thunder-carrots.md

```diff
@@ -0,0 +1,12 @@
+You dream of walking to the microwave in the kitchen.
+And you warm up a little soup. But then something happens.
+
+Out from the microwave, a dozen exploding thunder carrots swarm you smothering you as you cower.
+You fall to the ground and hit your head. You come to, and,
+
+Yes!
+
+That's how to [clone](https://www.google.com/search?q=fork+a+repo&oq=fork+a+repo&aqs=chrome..69i57j69i60j0l4.1958j0j1&sourceid=chrome&ie=UTF-8) a remote!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.